aws/vpc.tf

```diff
@@ -3,9 +3,9 @@ data "aws_availability_zones" "available" {
 
 module "vpc" {
   source  = "terraform-aws-modules/vpc/aws"
-  version = "2.47.0"
+  version = "3.14.0"
 
-  name                 = local.cluster_name
+  #name                 = local.cluster_name
```

Comment:
Any reason we don't want to name the vpc?

Reply:
It conflicts with the provider default tags and throws and error when the name is the same